clients/imodels-client-management/src/operations/imodel/IModelOperations.ts

```diff
@@ -235,14 +244,21 @@ export class IModelOperations<TOptions extends OperationOptions> extends Operati
     return result;
   }
 
-  protected getCreateEmptyIModelRequestBody(iModelProperties: IModelProperties): object {
-    return {
+  protected getCreateEmptyIModelRequestBody(iModelProperties: IModelProperties): IModelProperties {
+    const result: IModelProperties = {
       iTwinId: iModelProperties.iTwinId,
       name: iModelProperties.name,
       description: iModelProperties.description,
       extent: iModelProperties.extent,
-      containersEnabled: iModelProperties.containersEnabled
+      containersEnabled: iModelProperties.containersEnabled,
+      creationMode: iModelProperties.creationMode
     };
+
+    if (iModelProperties.geographicCoordinateSystem && iModelProperties.creationMode === "empty") {
+      result["geographicCoordinateSystem"] = iModelProperties.geographicCoordinateSystem;
```

Comment:
We should add gcs support to imodels-clients-authoring too.

Reply:
Now that `geographicCoordinateSystem` is returned from `getCreateEmptyIModelRequestBody`, I believe this support is there.

Reply:
Can you please modify some tests from imodels-clients-authoring to verify gcs setting really works? Just to be sure.